app/views/courses/show.html.erb

```diff
@@ -67,7 +71,9 @@
               <div
                 class="px-4 md:px-8 markdown-block markdown-block__permissive"
               >
-                <%= sanitize(presenter.about) %>
+              <div data-json-props="<%= presenter.markdown_prop(presenter.about) %>" class="convert-markdown">
+                <%= presenter.about %>
+              </div>
```

Comment:
@prashanth726 Shouldn't this wrapper `div` be rendered into only `if presenter.show_about?`

Reply:
@harigopal - We already checking for that in line [69](https://github.com/pupilfirst/pupilfirst/blob/579c3d2a16c08c92fb832fee885f59e1210a7a4e/app/views/courses/show.html.erb#L69)

Reply:
Got it.